lib/performance/AssetsOverSizeLimitWarning.js

```diff
@@ -0,0 +1,24 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Sean Larkin @thelarkinn
+*/
+var SizeFormatHelpers = require('../SizeFormatHelpers');
+
+function AssetsOverSizeLimitWarning(assetsOverSizeLimit, compilation, assetLimit) {
+	Error.call(this);
+	Error.captureStackTrace(this, AssetsOverSizeLimitWarning);
+	this.name = "AssetsOverSizeLimitWarning";
+	this.assets = assetsOverSizeLimit;
+
+	var assetLists = this.assets.map(function(asset) {
+		return "\n  " + asset.name;
+	}).join("");
+
+	this.message = "asset size limit: The following assets exceed the recommended size limit (" + SizeFormatHelpers.formatSize(assetLimit) + "). \n" +
```

Comment:
So in terms of output it would look like:

"Warning in asset size limit: <message>" Without affecting the other error formats I tried to stick with something that would work with `Warning in`. I'm open to suggestions. Overall I think an overhaul to the Error and Warnings are desirable. (Maybe separate PR?)